docs/structured-streaming-programming-guide.md

```diff
@@ -487,11 +487,11 @@ Dataset[Row] csvDF = spark
 spark = SparkSession. ...
 
 # Read text from socket 
-socketDF = spark \
-    .readStream()  \
-    .format("socket") \
-    .option("host", "localhost") \
-    .option("port", 9999) \
+socketDF = spark\
```

Comment:
Yeah I think it's good to standardize strings on double quotes, unless I'm missing some important difference. There are some other good clear fixes here. The whitespace changes are probably not worth it but I'd leave them in. What about this though, is it really more standard not to precede the continuation with a space?